src/components/Facility/FacilityForm.tsx

```diff
@@ -285,7 +353,47 @@ export default function CreateFacilityForm({
                     <Input
                       data-cy="facility-pincode"
                       placeholder="Enter pincode"
+                      maxLength={6}
+                      {...field}
+                      onChange={(e) => {
+                        field.onChange(e);
+                      }}
```

Comment:
it's the same as this right?
```suggestion
                      {...field}
```